projects/angular-notifier/jest.setup.ts

```diff
@@ -10,15 +10,17 @@ Object.defineProperty(window, 'getComputedStyle', {
 /**
  * ISSUE: https://github.com/angular/material2/issues/7101
  * Workaround for JSDOM missing transform property
+ * Commented this workaround out, as it is not needed anymore.
+ * Also it threw error `Cannot Redefine Property: transform`
  */
-Object.defineProperty(document.body.style, 'transform', {
-  value: () => {
-    return {
-      enumerable: true,
-      configurable: true,
-    };
-  },
-});
+// Object.defineProperty(document.body.style, 'transform', {
+//   value: () => {
+//     return {
+//       enumerable: true,
+//       configurable: true,
+//     };
+//   },
+// });
```

Comment:
if it's no longer needed, why are you commenting this code? maybe remove it?

Reply:
I wasn't sure about the working of this code, so didn't remove it at first.